examples/hello/Hello/Hello.fsproj

```diff
@@ -1,80 +1,25 @@
-<?xml version="1.0" encoding="utf-8"?>
-<Project ToolsVersion="15.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-  <Import Project="$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props" Condition="Exists('$(MSBuildExtensionsPath)\$(MSBuildToolsVersion)\Microsoft.Common.props')" />
+<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <Configuration Condition=" '$(Configuration)' == '' ">Debug</Configuration>
-    <Platform Condition=" '$(Platform)' == '' ">AnyCPU</Platform>
-    <SchemaVersion>2.0</SchemaVersion>
-    <ProjectGuid>4b0a4572-302e-4a58-b480-8b783c6bf520</ProjectGuid>
+    <TargetFrameworks>net462;net6.0</TargetFrameworks>
     <OutputType>Exe</OutputType>
```

Comment:
One minor issue I've found is that +x is not set on the produced executable on linux.

Reply:
I assume that you are talking only about `net462` target. `net6.0` set `-x` flag. Honestly I consider `net462` as legacy target which I do not want promote. I can understand that FStar users specifically can be very conservative, so I do not want rock the boat too much.

Reply:
It's not a big deal, and indeed it only affects the net462.